src/modules/lorem/index.ts

```diff
@@ -332,7 +358,26 @@ export class LoremModule {
 
     const method = this.faker.helpers.arrayElement(methods);
 
-    return `${this[method]()}`;
+    if (typeof options === 'number') {
+      options = { length: options };
+    }
+
+    if (options.length == null) {
+      return `${this[method]()}`;
+    }
+
+    const length = this.faker.helpers.rangeToNumber(options.length);
```

Comment:
Please use our standard way of decstructuring options:
```suggestion
    const { length: lengthOrRange } = options;

    if (lengthOrRange == null) {
      return `${this[method]()}`;
    }

    const length = this.faker.helpers.rangeToNumber(lengthOrRange);
```